core/src/main/scala/cats/Traverse.scala

```diff
@@ -55,6 +55,12 @@ import simulacrum.typeclass
       val G = Traverse[G]
     }
 
+  def composeFilter[G[_]: TraverseFilter]: TraverseFilter[λ[α => F[G[α]]]] =
```

Comment:
Are people happy with this? Since we only need a `Traverse` for `F` but a `TraverseFilter` for `G`, and there is a separate `compose` when `G` only has `Traverse`, it doesn't quite fit the common pattern.